packages/devextreme/js/ui/form.d.ts

```diff
@@ -61,6 +63,23 @@ export type FormItemType = 'empty' | 'group' | 'simple' | 'tabbed' | 'button';
 export type LabelLocation = 'left' | 'right' | 'top';
 /** @public */
 export type FormLabelMode = 'static' | 'floating' | 'hidden' | 'outside';
+/** @public */
+export type FormPredefinedButtonItem = 'reset' | 'submit' | 'smartPaste';
+
+/** @public */
+export type AIResult = Record<string, string | string[]>;
```

Comment:
This looks strange to me - I'm just worried there will be conflicts after we add more AI features - this "ai result" can be just a string or maybe even another type.

The type looks very general - shouldn't we reuse some type from aiIntegration core types? 🤔